charts/openmetadata/templates/deployment.yaml

```diff
@@ -2,21 +2,31 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "OpenMetadata.fullname" . }}
+  annotations:
+    {{- if .Values.annotations }}
```

Comment:
All the Kubernetes Objects should refer to this by calling
`{{- include "OpenMetadata.annotations" . | nindent 4 }}`.

Refactor to _helper.tpl as mentioned above in review.

Reply:
added {{- include "OpenMetadata.annotations" . | nindent 4 }}